controlpanel/api/models.py

```diff
@@ -85,39 +112,25 @@ def aws_delete_role(self):
 
     @property
     def _repo_name(self):
-        """Returns the repo name
-
-        The name is the part after the last slash in the URL, without
-        the '.git' (if present).
-
-        Examples:
-
-        "https://github.com/org/a_repo_name" => "a_repo_name"
-        "git@github.com:org/repo_2.git" => "repo_2"
-        """
-        repo_name = self.repo_url
+        return github_repository_name(self.repo_url)
 
-        if repo_name.endswith('/'):
-            repo_name = repo_name[:-1]
-
-        if repo_name.endswith('.git'):
-            repo_name = repo_name[:-4]
-
-        return repo_name.rsplit('/', 1)[1]
-
-    def get_customers(self):
-        return Auth0().get_group_members(group_name=self.slug)
+    @property
+    def customers(self):
+        return auth0.AuthorizationAPI().get_group_members(group_name=self.slug)
```

Comment:
#### 1d. Refactor Auth0 client module

> Lines 117 - 119

Our Auth0 client code was in a separate module `moj_analytics.auth0_client`, which was copied from another codebase. This code was too complicated - I wrote it 😞

[Next →]()